ovs/dal/hybrids/albanode.py

```diff
@@ -257,8 +261,11 @@ def _supported_osd_types(self):
         from ovs.dal.hybrids.albaosd import AlbaOSD
         if self.type == AlbaNode.NODE_TYPES.GENERIC:
```

Comment:
Perhaps you can store 
`<not_clients> = {NODE_TYPES.ASD: (ASDManagerClient, [allowed_types]),
                            NODE_TYPES.GENERIC: (GenericManagerClient, [allowed_types]),
                            NODE_TYPES.S3: (S3ManagerClient,[])}`
to prevent iterating over all client here as well, as you did on r94 - 97